scheduler/src/test/scala/com/sky/kms/base/KafkaIntSpecBase.scala

```diff
@@ -4,26 +4,29 @@ import com.sky.kms.kafka.Topic
 import com.sky.kms.utils.RandomPort.randomPort
 import eu.timepit.refined.auto._
 import net.manub.embeddedkafka.Codecs.{nullDeserializer, stringDeserializer}
-import net.manub.embeddedkafka.ConsumerExtensions.{ConsumerRetryConfig, _}
-import net.manub.embeddedkafka.{Consumers, EmbeddedKafka, EmbeddedKafkaConfig}
+import net.manub.embeddedkafka.{EmbeddedKafka, EmbeddedKafkaConfig}
 import org.apache.kafka.clients.consumer.{ConsumerRecord, KafkaConsumer}
 import org.apache.kafka.common.serialization.Deserializer
 import org.scalatest.WordSpecLike
 
-trait KafkaIntSpecBase extends EmbeddedKafka with WordSpecLike with Consumers {
+import collection.JavaConverters._
+import java.time.{Duration => JDuration}
+trait KafkaIntSpecBase extends EmbeddedKafka with WordSpecLike {
 
   implicit lazy val kafkaConfig = EmbeddedKafkaConfig(kafkaPort = randomPort(), zooKeeperPort = randomPort())
 
   val scheduleTopic: Topic      = "scheduleTopic"
   val extraScheduleTopic: Topic = "extraScheduleTopic"
 
-  val retryConfig = ConsumerRetryConfig(maximumAttempts = 50)
-
   def consumeFirstFrom[T : Deserializer](topic: String): ConsumerRecord[Array[Byte], T] =
-    withConsumer[Array[Byte], T, ConsumerRecord[Array[Byte], T]](_.consumeLazily(topic)(identity, retryConfig).head)
+    withConsumer[Array[Byte], T, ConsumerRecord[Array[Byte], T]] { cr =>
+      cr.subscribe(List(topic).asJavaCollection)
+      cr.poll(JDuration.ofSeconds(60)).iterator().next()
```

Comment:
Just to give the 60 a name how about something like:

```scala
  def consumeFirstFrom[T : Deserializer](topic: String, timeout: Duration = 60.seconds): ConsumerRecord[Array[Byte], T] =
    withConsumer[Array[Byte], T, ConsumerRecord[Array[Byte], T]] { cr =>
      cr.subscribe(List(topic).asJavaCollection)
      cr.poll(timeout).iterator().next()
```

Same for the `consumeSomeFrom` function

Reply:
Done